editor/js/Sidebar.Settings.Shortcuts.js

```diff
@@ -95,13 +95,13 @@ function SidebarSettingsShortcuts( editor ) {
 
 	}
 
-	document.addEventListener( 'keydown', function ( event ) {
+	function shortcutsListener( event ) {
 
 		switch ( event.key.toLowerCase() ) {
 
 			case 'backspace':
 
-				event.preventDefault(); // prevent browser back
+				// event.preventDefault(); // prevent browser back -- this is no longer an issue since 2016
 
 				// fall-through
```

Comment:
If we're not using this case, can it be removed then?

Reply:
That was my first idea. But then I thought what if somebody does use this. I tried to remove all event listeners on modal close -- to no avail.